yarn-project/pxe/src/entrypoints/client/lazy/utils.ts

```diff
@@ -0,0 +1,53 @@
+import { BBWASMLazyPrivateKernelProver } from '@aztec/bb-prover/wasm/lazy';
+import { createLogger } from '@aztec/foundation/log';
+import { createStore } from '@aztec/kv-store/indexeddb';
+import { LazyProtocolContractsProvider } from '@aztec/protocol-contracts/providers/lazy';
+import { WASMSimulator } from '@aztec/simulator/client';
+import type { AztecNode } from '@aztec/stdlib/interfaces/client';
+
+import type { PXEServiceConfig } from '../../../config/index.js';
+import { PXEService } from '../../../pxe_service/pxe_service.js';
+import type { PXECreationOptions } from '../pxe_creation_options.js';
+
+/**
+ * Create and start an PXEService instance with the given AztecNode.
+ * Returns a Promise that resolves to the started PXEService instance.
+ *
+ * @param aztecNode - The AztecNode instance to be used by the server.
+ * @param config - The PXE Service Config to use
+ * @param
+ * @returns A Promise that resolves to the started PXEService instance.
+ */
+export async function createPXEService(
```

Comment:
This is a duplicate of the 'bundle' version except it imports the lazy prover no? is that to be expected?

Reply:
Yes. Since imports happen "automatically" we need to keep these separate even if fairly similar